### PR TITLE
Update release-binary script from tezos-packaging

### PR DESCRIPTION
**Pull request by balsoft**
Update `release-binary` script so that it uploads all the artifacts separately.

### CODE REVIEW

File: scripts/release-binary.sh [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-#!nix-shell -p gitAndTools.hub git -i bash
-
-# This script will create a new GitHub (pre-)release for the current repository with tag `auto-release`.
-# If such tag already exists, it will be overwritten.
-# It will automatically build `release/default.nix` and put its result into a tar archive that will be attached to the release.
-
-# Suggested usage:
-# 1. Make sure your repository has `release/default.nix` that produces whatever you want to release.
-# 2. Run this script from CI on each commit to master.
-# 3. When you think it's time to make a real release, edit the auto release: change the tag, write a description, remove "pre-release" checkmark.
```
Comment (gromakovsky) [inline scripts/release-binary.sh]: Why did you remove the instructions btw?

File: scripts/release-binary.sh [hunk 2]
```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -p gitAndTools.hub git -i bash
-
-# This script will create a new GitHub (pre-)release for the current repository with tag `auto-release`.
-# If such tag already exists, it will be overwritten.
-# It will automatically build `release/default.nix` and put its result into a tar archive that will be attached to the release.
+#!nix-shell -p gitAndTools.hub git rename -i bash
```
Comment (zhenyavinogradov) [inline scripts/release-binary.sh]: I think `rename` is not used

File: scripts/release-binary.sh [hunk 3]
```diff
@@ -13,29 +9,38 @@
...
+    rm -rf "$TEMPDIR"
 }
 trap finish EXIT
 
+assets_dir=$TEMPDIR/assets
 
-# Build release/default.nix
-nix-build release -o $TEMPDIR/$project
+# Build release.nix
+nix-build . -A release -o "$TEMPDIR"/"$project" --arg timestamp "$(date +\"%Y%m%d%H%M\")"
```
Comment (zhenyavinogradov) [inline scripts/release-binary.sh]: Usage instructions above mention `release/default.nix`, but here `default.nix` is used

File: scripts/release-binary.sh [hunk 4]
```diff
@@ -13,29 +9,38 @@
...
 # Update the tag
 git fetch # So that the script can be run from an arbitrary checkout
 git tag -f auto-release
 git push --force --tags
 
+# Combine all assets
+assets=()
+for file in $assets_dir/*; do
+    echo $file
+    assets+=("-a" $file)
```
Comment (zhenyavinogradov) [inline scripts/release-binary.sh]: `"$file"` should be quoted

**Review by zhenyavinogradov — CHANGES_REQUESTED** [review]

File: scripts/release-binary.sh [hunk 5]
```diff
@@ -13,29 +9,38 @@
...
-  rm -rf "$TEMPDIR"
+    rm -rf "$TEMPDIR"
 }
 trap finish EXIT
 
+assets_dir=$TEMPDIR/assets
 
-# Build release/default.nix
-nix-build release -o $TEMPDIR/$project
+# Build release.nix
```
Comment (zhenyavinogradov) [inline scripts/release-binary.sh]:
```suggestion
# Build release/default.nix
```

**Review by zhenyavinogradov — CHANGES_REQUESTED** [review]

**Review by zhenyavinogradov — APPROVED** [review]

**Pull request merged**